OHAlertView/OHAlertView.m

```diff
@@ -104,6 +104,15 @@ -(instancetype)initWithTitle:(NSString *)title
 	return self;
 }
 
+- (void)dealloc
+{
+    #if USE_UIALERTVIEW == 0
```

Comment:
I feel it would be more logical/readable for it to be written as `#if ! USE_UIALERTVIEW` instead, right?